spec/features/commands_spec.rb

```diff
@@ -157,6 +157,18 @@
     end.to change { deployment_requests.count }.by(1)
   end
 
+  scenario 'checking a locked branch' do
+    # It says that it's locked after someone locks it.
+    command "/deploy lock staging on acme-inc/api: I'm testing some stuff", as: slack_accounts(:david)
+    command '/deploy check lock staging on acme-inc/api', as: slack_accounts(:david)
```

Comment:
What do you think if we just generalize this as a `/deploy check <environment> on <repo>` command, and then include the lock information in some attachments? Couple reasons:

1. It's less to type.
2. We can add more information later on, like what ref is currently deployed to the environment, who was the last deploy, etc.